media/distribution.html

```diff
@@ -19,14 +19,28 @@ <h2>Well-deployed technologies</h2>
+          <p>Media providers also need some mechanism to assess the user's perceived playback quality in order to alter the quality of content transmitted using adaptive streaming. The <a data-featureid="media-playback-quality">Media Playback Quality</a> specification, initially part of MSE, exposes metrics on the number of frames that were displayed or dropped.</p>
+        </div>
+        <div data-feature="Adaptive streaming">
+          <p>A media stream may be composed of different programs that use different encodings, or may need to embed ads that have different characteristics. Applications cannot easily and smoothly merge these heterogeneous sub-streams back into one stream using the first version of Media Source Extensions (MSE), as it does not support this use case. Content providers need to work around this limitation, e.g. transcoding and re-packaging the ad content to be consistent with the program content prior to distribution where possible. The Media Working Group has started work on a <a data-featureid="mse-v2/codec-switching">codec switching feature for MSE</a> to address this issue.</p>
+        </div>
+        <div data-feature="Realtime distribution">
+          <p>When broadcasting popular live events on an IP network, content distribution should introduce as little latency as possible. <a data-featureid="p2p">WebRTC</a> technologies may be used to reach sub-second latency. Additionally, these technologies may be used to create a peer-to-peer content delivery network and spread the load of video distribution among watchers. Extensions to the base WebRTC specification allow Web applications to further refine media parameters:</p>
+          <ul>
+            <li><a data-featureid="mst-content-hint">Content Hints</a> lets Web application advertise the type of media content that is being consumed (e.g. speech or music, movie or screencast) so that user agents may optimize encoding or processing parameters;</li>
+            <li><a data-featureid="webrtc-svc">Scalable Video Coding</a> (SVC) allows Web application to leverage SVC (whereby subset video streams can be derived from the larger video stream by dropping packets to reduce bandwidth consumption), making providing video at different qualities to multiple destinations with the same initial video stream easier.</li>
+          </ul>
+        </div>
+        <div data-feature="Media packaging">
```

Comment:
I'm not sure if this is useful enough, but EPUB is also a package format and supports multimedia. (See [1](https://www.w3.org/publishing/epub3/epub-overview.html#sec-multimedia) / [2](https://github.com/IDPF/epub3-samples/tree/master/30/cc-shared-culture/EPUB) for examples.)

Reply:
Good point, the goal here (and even more so in the "Features not covered by ongoing work") was indeed to point out that different communities are looking at this problem space from different angles and that there may be a need for convergence. I'll add the Digital Publishing perspective (perhaps in the "not covered" section)